src/App.js

```diff
@@ -1,25 +1,31 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import { Routes, Route, Link } from 'react-router-dom';
+import AddBooks from './components/AddBooks';
+import RemoveBook from './components/RemoveBook';
+import Catergories from './components/Catergories';
+import DisplayBooks from './components/DisplayBooks';
 
-function App() {
+function BookStore() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <h1>Book Store</h1>
+      <ul>
+        <li>
+          <Link to="/books">Books</Link>
+        </li>
+        <li>
+          <Link to="/catergories">Catergories</Link>
+        </li>
+      </ul>
+      <hr />
+      <Routes>
+        <Route exact path="/books" element={<DisplayBooks />} />
+        <Route path="/catergories" element={<Catergories />} />
+      </Routes>
+      <AddBooks />
+      <RemoveBook />
```

Comment:
- [ ] **Required:** These two components should be rendered on the `DisplayBooks` component and should only be visible on the books page.

Reply:
Hi, @brytebee Thank you for the review, however, that is not a requirement in this project, so please explain what makes this more efficient to do, more than my current way of doing it? 

All the best-K

Reply:
- [ ] As requested by the previous reviewer, make sure that you render those two components in the `DisplayBooks` component. For now, we can see the form for adding and removing books on the `home` page and on the `categories` page but, it should only see on the homepage. 
- To implement this change you can only remove those two components from your `App.js` component, which means from this file with the function Bookstore, and then add them in the `DisplayBooks.js` component like in the screenshot below:
![image](https://user-images.githubusercontent.com/71874425/177817575-3d9cd481-fe2f-4d48-9713-5b939c21df20.png)